### PR TITLE
fix for light forms not saving correctly

### PR DESCRIPTION
**Pull request by Ryan-rsm-McKenzie**

### CODE REVIEW

**Comment by ryobg** [comment]
Thanks for the contribution (#60)! How this was tested? I more or less get the idea of the change and apart from minor stylistic issue (which I can fix on my own) seems okay.

File: src/JContainers/src/skse/skse.cpp [hunk 1]
```diff
@@ -158,11 +158,9 @@ struct real_api : public skse_api
 
     FormId resolve_handle (FormId handle) override
     {
-        std::uint64_t out = 0,
-                      in  = static_cast<std::uint64_t> (handle);
-        if ((in & 0xFF000000u) == 0xFF000000u)
-            return handle; // dynamic form - return untouched
-        return g_serialization->ResolveHandle (in, &out) ? (FormId) out : FormId::Zero;
+		auto oldID = static_cast<UInt32>(handle);
```
Comment (ryobg) [inline src/JContainers/src/skse/skse.cpp]: Truncating 64 to 32, is this the way?

**Comment by Ryan-rsm-McKenzie** [comment]
`FormId` is defined as 32bits, we're not losing anything here.

File: src/JContainers/src/skse/skse.cpp (hunk 1 above)
Comment (ryobg) [reply src/JContainers/src/skse/skse.cpp]: Hmm... yes, true. Thats what you get when mixing types and names. `FormId` is 32-bit.

**Comment by ryobg** [comment]
So @Ryan-rsm-McKenzie , how did you test that? Before giving it to everybody else I need to know how reliable is this (I still cant test it regretfully, as I'm away from my machine).

**Comment by Ryan-rsm-McKenzie** [comment]
I handed it to duncan who's the author of [iEquip](https://www.nexusmods.com/skyrimspecialedition/mods/27008) and he confirmed it fixed the issues he was having with resolving light forms

**Pull request merged**